documentation/metadata-editing.md

```diff
@@ -0,0 +1,87 @@
+# Editing metadata of resources
+
+## What is a metadata resource
+
+Metadata resources contain auxiliary information about a resource. This indicates that a resource will always have a corresponding metadata resource and therefore metadata resources can not be created or deleted directly.
```

Comment:
Make the lines shorter so there is no word wrap in the editor, similar to how the other documentation files are written.